Traceback (most recent call last):
  File "/home/dat-asset-235/Documents/djangoOscar/oscar_proj/oscar_proj/settings.py", line 196, in <module>
    JamAuthTokenList = JamAuthToken.split('|')
AttributeError: 'NoneType' object has no attribute 'split'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/core/management/__init__.py", line 325, in execute
    settings.INSTALLED_APPS
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 57, in __getattr__
    self._setup(name)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 107, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/dat-asset-235/Documents/djangoOscar/oscar_proj/oscar_proj/settings.py", line 202, in <module>
    """)
Exception: 
    Please setup JamAuthToken with "export JAM_AUTH_TOKEN='.......|.....-....-......-....-.....'"
    from terminal.
    
Traceback (most recent call last):
  File "/home/dat-asset-235/Documents/djangoOscar/oscar_proj/oscar_proj/settings.py", line 196, in <module>
    JamAuthTokenList = JamAuthToken.split('|')
AttributeError: 'NoneType' object has no attribute 'split'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/core/management/__init__.py", line 325, in execute
    settings.INSTALLED_APPS
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 57, in __getattr__
    self._setup(name)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/site-packages/django/conf/__init__.py", line 107, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/home/dat-asset-235/Documents/djangoOscar/OscarEnv/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/dat-asset-235/Documents/djangoOscar/oscar_proj/oscar_proj/settings.py", line 202, in <module>
    """)
Exception: 
    Please setup JamAuthToken with "export JAM_AUTH_TOKEN='.......|.....-....-......-....-.....'"
    from terminal.
    
